synapse/storage/schema/__init__.py

```diff
@@ -75,6 +75,7 @@
 Changes in SCHEMA_VERSION = 72:
     - event_edges.(room_id, is_state) are no longer written to.
     - Tables related to groups are dropped.
+    - Remove unstable private read receipts
```

Comment:
We only include things here if they change the schema, which this doesn't. This only affects data in it.